feeds/public.txt

```diff
@@ -67,6 +67,7 @@ fluid-framework
 @fluidframework/core-interfaces
 @fluidframework/container-definitions
 @fluid-internal/client-utils
+@fluid-experimental/tree-react-api
```

Comment:
Feed files doing their job. 👍🏻 You can be confident that the package will wind up on npm when we release.